lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -1974,7 +1976,8 @@ struct AggOneShot : public mlir::RewritePattern {
 
   SmallVector<Value> getCompleteWrite(Operation *lhs) const {
     auto lhsTy = lhs->getResult(0).getType();
-    if (!isa<BundleType, FVectorType>(lhsTy))
+    if (!type_isa<BundleType, FVectorType>(lhsTy) ||
```

Comment:
I think this `isPassive` is redundant with the change made in #5651 ?

I went back-and-forth whether this should be handled here or there (and IIRC gathering the connect ops while visiting here instead of re-discovering them in the rewriter?) but anyway unless I'm missing something only need the one check or the other?